cypress/integration/collections.js

```diff
@@ -70,14 +64,13 @@ describe("Collections", () => {
     cy.visit(urlList.collections);
     let collection;
 
-    collectionsSteps
-      .createCollection(collectionName, false, defaultChannel)
+    createCollection(collectionName, false, defaultChannel)
```

Comment:
If every test need collection, you can create it in `beforeEach`

Reply:
These tests are creating collections with different options. First test is creating not published collection, second published, another one with different channel. I can move it to before each, and in test update created collection, but I don't think it's a good idea.